vagga.yaml

```diff
@@ -0,0 +1,25 @@
+containers:
+    gdg_dev:
+      setup:
+      - !Ubuntu trusty
+      - !UbuntuUniverse
+      - !Install [software-properties-common, npm, curl, git, openssl, libssl-dev, libpq-dev, ca-certificates, build-essential, wget, python-pip, python-dev, mercurial]
+      - !AptTrust keys: [5BB92C09DB82666C]
+      - !UbuntuPPA fkrull/deadsnakes
+      - !Install [python3.5]
+      - !Py3Requirements requirements/dev.txt
+      - !NpmInstall [nodejs, bower]
+      volumes:
+          /root: !BindRW /volumes/home
+          /usr/local/: !BindRW /volumes/home
+          /var: !BindRW /volumes/home
+
+commands:
+  run: !Supervise
+    description: Run app - testing status
+    children:
+      env: !Command
+        container: gdg_dev
+        run: |
+          cd gdg.org.ua
+          sudo make dev-deps
```

Comment:
I'd move that to setup phase. You need to run app itself here. I think, it's just `make run-dev`